src/renderers/server-render.jsx

```diff
@@ -31,15 +31,9 @@ const DOCTYPE = '<!DOCTYPE html>';
  * @module ServerRender
  */
 
-function getHtml(baseUrl, assetPublicPath, clientFilename, initialState={}, appMarkup='') {
+function getHtml(props) {
```

Comment:
definitely an overdue change - instead of opaque `props` object however, we could eliminate another layer of abstraction by just accepting a `<Dom>` instance - it's easy enough for the caller to write `getHtml(<Dom {...props}>)` and the actual values are not used in the `getHtml` function - it also makes the `getHtml` call a bit more self-explanatory while reducing the need for redundant code documentation